edx_api/integration_test.py

```diff
@@ -395,3 +396,28 @@ def test_user_info_timeout():
         get.side_effect = mocked_timeout
         with pytest.raises(Timeout):
             user_info.get_user_info()
+
+
+@require_integration_settings
+def test_enrollments_list():
+    """
+    Enrolls the user in a course and then pulls down the enrollments for the user.
+    This assumes that the course in the edX instance is available for enrollment.
+    """
+    api = EdxApi({'access_token': ACCESS_TOKEN}, base_url=BASE_URL)
+    enrollments = api.enrollments.get_enrollments()
+
+    cnt = 0
+    for enrollment in enrollments:
+        assert enrollment.course_id
+        assert enrollment.created
+        assert enrollment.mode
+        assert enrollment.is_active
+        assert enrollment.user
+        if enrollment.mode != u'verified':
```

Comment:
Are these meant to be checking the same thing?

Reply:
Ah, yes. Typo on my side. Thanks.

Reply:
Gotcha, thanks :)